ansible/restund.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: gather tls certificate
+  hosts: localhost
+  pre_tasks:
+    - name: Read TLS config
+      set_fact:
+        # Reminder that the pem file should look like:
+        # -----BEGIN CERTIFICATE-----
+        # --- ... CERT CONTENT ... --
+        # -----END CERTIFICATE-----
+        # -----BEGIN CERTIFICATE-----
+        # --- ... INTERMEDIATE ..----
+        # -----END CERTIFICATE----
+        # -----BEGIN PRIVATE KEY-----
+        # --- .... PRIV KEY -----
+        # -----END PRIVATE KEY-----
+
+        restund_tls_certificate: "{{ lookup('file', '/tmp/tls_cert_and_priv_key.pem') }}"
```

Comment:
You can move this line to the `hosts: restund` block and delete the localhost block entirely. `lookup` plugins are executed on localhost by default.

Reply:
Oh, OK. Thanks!